temporal/t.rast.export/t.rast.export.py

```diff
@@ -37,9 +37,9 @@
 
 # %option G_OPT_M_DIR
 # % key: directory
-# % description: Path to the work directory, default is /tmp
+# % label: Path to the temporary working directory
```

Comment:
No really strong opinion, but since this is the export directory why not saying just that, i.e., `Path to the export directory`? Temporary working seems a bit odd to me.

Reply:
I think what is also expected is that the `./` stuff is not in the default answer but resolved in the program internally. It would have to be `.` and not `./` anyway.